### ISSUE TITLE
Lock resizing during cropping image

### ISSUE THREAD

**Issue by Gosten** [hsbijarniya/image_editor_plus#63]
Hello,

I was wondering if it's possible to add fixed size functionality to image crop?
Right now I can set fixed aspect ratio, but I can resize it. 

I wonder if it would be possible to lock crop area from resizing?

**Comment by Tonalga69** [comment]
I agree, it would be great if it shows a loading widget to let know users that the image is cropping
